movies/movies

```diff
@@ -89,13 +91,18 @@ printMovieInfo()
   echo '=================================================='
   echo "| Title: $title"
   echo "| Year: $year"
-  if [[ $imdbScore != "" ]];then echo "| IMDB: $imdbScore";fi
-  if [[ $tomatoScore != "" ]];then echo "| Tomato: $tomatoScore";fi
-  if [[ $rated != "N/A" && $rated != "" ]];then echo "| Rated: $rated";fi
+  echo "| Runtime: $runtime"
+  if [[ $imdbScore != "" ]];then echo "| IMDB: $imdbScore"; fi
+  if [[ $tomatoScore != "" ]];then echo "| Tomato: $tomatoScore"; fi
+  if $detail && [[ $metacriticScore != "" ]];then echo "| Metascore: $metacriticScore"; fi 
+  if [[ $rated != "N/A" && $rated != "" ]];then echo "| Rated: $rated"; fi
   echo "| Genre: $genre"
   echo "| Director: $director"
   echo "| Actors: $actors"
   if [[ $plot != "N/A" && $plot != "" ]];then echo "| Plot: $plot"; fi
+  if $detail && [[ $$boxOffice != "" ]]; then echo "| Box Office: $boxOffice"; fi
```

Comment:
Can ```$detail``` go inside the test expression? Like ``` if [[ $detail && $boxOffice != "" ]]``` Your implementation works it just is a little harder for me to read.